src/client/dfuse/dfuse_main.c

```diff
@@ -158,9 +172,11 @@ main(int argc, char **argv)
 			dfuse_info->di_foreground = true;
 			break;
 		case 'h':
+			show_help(argv[0]);
 			exit(0);
 			break;
 		case '?':
+			show_help(argv[0]);
 			exit(1);
```

Comment:
Why exit(1) here ?

Reply:
This is the code-path taken where another option is specified that isn't understood, so printing the help and exiting is the correct thing here.